security/org-policies/policies/RestrictivePolicy.json

```diff
@@ -106,12 +106,8 @@
         {
             "Sid": "DenyOrgRoleModification",
             "Effect": "Deny",
-            "Action": [
-                "iam:*"
-            ],
-            "Resource": [
-                "arn:aws:iam::*:role/OrgRole"
-            ]
+            "NotAction": ["iam:GetRole"],
```

Comment:
@avnes This might block more than the original `iam:*` actions (`sts:*`), so I'm not really sure about using this `NotAction`.

Could we come up with a list of actions that we want to explicitly deny here instead of the original `iam:*` since the intent is to only prevent modification of the role?